test/integration_test.coffee

```diff
@@ -5,27 +5,28 @@ assert = require 'assert'
 provider_key = process.env.TEST_3SCALE_PROVIDER_KEY
 application_key = process.env.TEST_3SCALE_APP_KEY
 application_id = process.env.TEST_3SCALE_APP_ID
+service_id = process.env.TEST_3SCALE_SERVICE_ID
```

Comment:
This should be set in the `.travis.yml` file with the rest of envs: https://github.com/3scale/3scale_ws_api_for_nodejs/blob/master/.travis.yml

It should be encrypted like the provider_key: https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml